charts/mimir-distributed/templates/gateway/gateway-dep.yaml

```diff
@@ -0,0 +1,115 @@
+{{- if .Values.enterprise.enabled -}}
```

Comment:
Do we need a gateway and an nginx? What features is the gateway currently providing that nginx isn't? An option here could just be changing the value of the `nginx.conf` based on whether or not enterprise is enabled (ie. to add the `/admin` routes). WDYT?